components/card/card.css

```diff
@@ -0,0 +1,70 @@
+.card-box {
+    width: 25rem;
+    padding: 1rem;
+    box-shadow: 0px 0px 4px 1px var(--bg-secondary-color);
+}
+
+.card-title {
+    color: var(--primary-color);
+    font-size: 2rem;
+    font-weight: bolder;
+    margin-bottom: 1rem;
+}
+
+.card-content {
+    font-size: 1.5rem;
+}
+
+.card-head-box {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    position: relative;
+}
+
+.card-head-box a {
+    display: flex;
+    text-decoration: none;
+}
+
+.card-head-box a i {
+    padding: .4rem;
+    border-radius: 50%;
+    text-align: center;
+    color: var(--text-color);
+    cursor: pointer;
+}
+
+.card-head-box img {
+    overflow: hidden;
+}
+
+.txt-overlay {
+    position: absolute;
+    bottom: .1rem;
+    padding-left: .5rem;
+}
+
+.card-box .bx {
+    position: absolute;
+    top: .3rem;
+    right: .2rem;
+    font-weight: bolder;
+    color: var(--secondary-color);
+}
+
+.card-head-box a i:hover {
+    background-color: var(--bg-secondary-color);
+    color: var(--text-color);
+}
+
+.card-head-box .badge {
+    padding: .2rem 1rem;
+    position: absolute;
+    font-size: 1rem;
+    color: var(--secondary-color);
+    background-color: var(--primary-color);
+    top: -1rem;
+    left: -1rem;
```

Comment:
A suggestion here would be that you could sort all these properties in alphabetical order as that helps in debugging as well as code maintainability.

Reply:
Okay, noted